bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/Layer/LayerTools.jsx

```diff
@@ -62,22 +61,25 @@ const LayerTools = ({ model, controller }) => {
     const map = Oskari.getSandbox().getMap();
     const reasons = !map.isLayerSupported(model) ? map.getUnsupportedLayerReasons(model) : undefined;
     const reason = reasons ? map.getMostSevereUnsupportedLayerReason(reasons) : undefined;
-    const statusOnClick = backendStatus.status !== 'UNKNOWN' ? () => controller.showLayerBackendStatus(model.getId()) : undefined;
+    let layerIcon;
+    if (model.hasTimeseries()) {
+        layerIcon = (
+            <Tooltip title={<Message messageKey="layer.tooltip.timeseries" />}>
+                <TimeSerieIcon />
+            </Tooltip>
+        );
```

Comment:
I'll think on this a bit before merging. I think timeseries shouldn't have a special handling here but should be part of the LayerStatus tag, but it might mean some other changes a bit. I'll get back to this tomorrow probably.

Reply:
Yeah, I had the same feeling too

Reply:
Let's remove the timeseries special handling from here. Then the icon comes from LayerIcon.jsx which handles timeseries with the other change you made. You'll need to dig out that hasTimeSeries() prop inside LayerStatus so it's passed to LayerIcon though.